package.json

```diff
@@ -53,58 +35,53 @@
     "!**/*.d.ts"
   ],
   "dependencies": {
-    "cliui": "^8.0.1",
+    "cliui": "^9.0.1",
     "escalade": "^3.1.1",
     "get-caller-file": "^2.0.5",
     "require-directory": "^2.1.1",
-    "string-width": "^4.2.3",
+    "string-width": "^7.2.0",
     "y18n": "^5.0.5",
     "yargs-parser": "^21.1.1"
   },
   "devDependencies": {
-    "@rollup/plugin-terser": "^0.4.4",
-    "@rollup/plugin-typescript": "^12.1.2",
+    "@babel/eslint-parser": "^7.26.10",
+    "@babel/preset-typescript": "^7.26.0",
     "@types/chai": "^4.2.11",
     "@types/mocha": "^9.0.0",
     "@types/node": "^20.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.26.1",
     "browserslist-generator": "^2.0.1",
     "c8": "^9.1.0",
     "chai": "^4.2.0",
     "chalk": "^4.0.0",
     "concurrently": "^7.6.0",
```

Comment:
Not currently using `concurrently` so can be removed.

Reply:
(To be clear, "Not currently using `concurrently`" if removed from npm run script for `watch`.)